src/containers/Downloads/DataSet/index.js

```diff
@@ -198,6 +196,34 @@ let DataSetErrorDownloading = ({
                 Try Again
               </Button>
             )}
+
+            <p>
+              If the problem persists, please contact{' '}
+              <a href="mailto:ccdl@alexslemonade.org" className="link">
+                ccdl@alexslemonade.org
+              </a>
+              {dataSet.failure_reason && (
+                <span>
+                  {' '}
+                  or{' '}
+                  <a
+                    href="https://github.com/AlexsLemonade/refinebio/issues/new"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    className="link"
+                  >
+                    file a ticket on Github
```

Comment:
Maybe this line shoud be changed to
```
report the issue to us
```
("file a ticket on GitHub" sound so techy and even scary to a user who never uses github).